nipype/pipeline/plugins/multiproc.py

```diff
@@ -128,6 +135,10 @@ def __init__(self, plugin_args=None):
         self._task_obj = {}
         self._taskid = 0
 
+        # Cache current working directory and make sure we
+        # change to it when workers are set up
+        self._cwd = os.getcwd()
```

Comment:
Is this the workflow base directory or the CWD of the shell? Could this cause things to dump into the local directory?

Reply:
It shouldn't because interfaces handle their WD. I think this is fixing an edge case for fmriprep where we are spinning up and killing workers all the time.

Reply:
Ok it turns out that some `SimpleInterface`s are writing to the workflow base directory :(

Reply:
Well that's... suboptimal.

Reply:
It's actually bothering, because it means that without this patch, those interfaces are being run in some other unexpected path.

Reply:
Apparently it is a problem only with `SimpleInterface`